### PR TITLE
style.css and index.html file modified

### PR DESCRIPTION
**Pull request by eyoab01**
changes made on Mobile desktop version.
Thank you.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -188,6 +188,7 @@ <h2>Multi-Post Stroies</h2>
         </div>
       </div>
     </section>
+  </div>
```
Comment (christianbiring1) [inline index.html:191]: - [ ] Kindly make sure you have the form on your page. As you can there's no one on your page. :point_down: 
image
 - [ ] Figma design :point_down: 
 
image

File: index.html [hunk 2]
```diff
@@ -55,7 +55,7 @@ <h1>I'm Tesfaye <br>Glad to see you!</h1>
         </ul>
       </section>
     </main>
-
+<div class="card-container">
```
Comment (christianbiring1) [inline index.html:58]: - [ ] Kindly make sure your card_works are well positioned as the ones on the template in Figma.
Your design :point_down: 
image
 Figma design :point_down: 
image

File: style.css [hunk 3]
```diff
@@ -109,19 +109,11 @@ a {
...
 .card {
   display: flex;
-  flex-direction: column;
-  max-width: 330px;
+  flex-direction: row;
+  align-self: center;
+  max-width: 1155px;
   background: #fff;
   border: 2px solid #dfe1e6;
   border-radius: 16px;
```
Comment (christianbiring1) [inline style.css]: - [ ] Kindly make sure your cards are responsive and look similar to the ones provided in Figma.
Your mobile version :point_down: 
image
Figma design :point_down: 
image

**Review by christianbiring1 — CHANGES_REQUESTED** [review]
Hi @eyoab01 , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.
# STATUS: REQUEST CHANGES :recycle: 
163947610-e4d849d5-e69c-49b9-90bd-a4191f4e2856

 ## Highlights
- [ ] Linters checks are all green :heavy_check_mark: 

 ## Required Changes ♻️
- [ ] Kindly make sure your project is documented in a professional way. There must be the `title` of the project, the `description` which detailed what the project is about
- [ ] Kindly make sure you have a descriptive summary in your PR telling the changes that were introduced.
image
### Kindly make sure you don't open another pull request for the next review
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Alfredbis29 — COMMENTED** [review]
Hi @eyoab01 , 

While you made a great effort in this project, unfortunately, I cannot proceed to review your code.

## Invalid Code Review Request

You have submitted a project, that you didn't do any changes from the previous review , I suggest you to request for another opion once you are ready , please follow all requirements from the previous review. :wink: 

Your Code Review Request will be marked as invalid in your Dashboard, so please submit a new one once you are ready 🙏

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

--------

_Invalid Code Review Request **does not count** into the code reviews limit._

**Comment by eyoab01** [comment]
This one is my final, i cant submit on student dashboard.

**Review by Olamarx — APPROVED** [review]
Hi @eyoab01, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

- [ ] As said by the previous reviewer, kindly always add a description to your PR

## Highlights
- Good commit messages ✔️
- Descriptive readme 👍🏼 
- Good use of Git and GitHub flow ✅ 
- All linters passed ✔️ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**